.github/workflows/integration_tests.yml

```diff
@@ -11,8 +13,8 @@ on:
       - prod
 
 jobs:
-  integration_tests:
-    name: Integration tests
+  integration_tests_quality:
```

Comment:
This job is run for all PR. Ideally, it should be triggered only when changes are made in the `/tests` folder. The easy way to do so is to have another workflow. Maybe it's possible to configure it in a job.